packages/wrangler/templates/d1-beta-facade.js

```diff
@@ -238,4 +238,11 @@ var shim_default = {
 		return worker.trace(traces, getMaskedEnv(env), ctx);
 	},
 };
-export { shim_default as default };
+var maskDurableObjectDefinition = (cls) =>
+	class extends cls {
+		constructor(state, env) {
+			super(state, getMaskedEnv(env));
+		}
+	};
+__DO_REEXPORTS__;
+export { shim_default as default, maskDurableObjectDefinition };
```

Comment:
Does `maskDurableObjectDefinition` need exported?

Reply:
Only way I could stop `esbuild` from blowing away `maskDurableObjectDefinition` when the shim gets built (since it looks like dead code to esbuild)